packages/vscode-extension/lib/network/AsyncBoundedResponseBuffer.test.ts

```diff
@@ -79,15 +79,15 @@ describe("AsyncBoundedResponseBuffer", () => {
     const result1 = await buffer.put(requestId1, Promise.resolve(responseData1));
     const result2 = await buffer.put(requestId2, Promise.resolve(responseData2));
     assert.strictEqual(result1, true);
-    assert.strictEqual(result2, false);
+    assert.strictEqual(result2, true);
 
     const retrievedPromise1 = buffer.get(requestId1);
     const retrievedPromise2 = buffer.get(requestId2);
     assert(retrievedPromise1 !== undefined);
-    assert(retrievedPromise2 === undefined);
+    assert(retrievedPromise2 !== undefined);
 
-    assert.strictEqual(result1, buffer.get(requestId1));
-    assert.strictEqual(result2, buffer.get(requestId2));
+    assert.strictEqual(undefined, buffer.get(requestId1));
+    assert.strictEqual(undefined, buffer.get(requestId2));
```

Comment:
The convention is to write `assert.strictEqual(actual_value, expected_value)`

```suggestion
    assert.strictEqual(buffer.get(requestId1), undefined);
    assert.strictEqual(buffer.get(requestId2), undefined);
```